src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,45 @@
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { SearchLink } from './SearchLink';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const centuries = [16, 17, 18, 19, 20];
```

Comment:
Should be declared outside the component, let's assume we are taking this value from environmental variables and declare it in uppercase.